runtime/src/bank.rs

```diff
@@ -1447,6 +1451,9 @@ mod tests {
 
             // works iteration 0, no-ops on iteration 1 and 2
             bank.squash();
+
+            assert_eq!(parent.transaction_count(), 1);
```

Comment:
This assertion fails.  parent.transaction_count() is actually 0.   The bug appears to be in accounts squash.
@sambley - would you be able to take a look at this?

Reply:
Fixed as part of https://github.com/solana-labs/solana/pull/3000